### ISSUE TITLE
Ready-only policies not supported by Block drivers

### ISSUE THREAD

**Issue by ehmry** [genodelabs/genode#2469]
In some situations (perhaps most if embedded Genode appliances proliferate) it is vital enforce read-only access to boot media. I suggest the ``Block::Session`` class optionally enforce ready-only ``Block`` sessions by policy.

**Comment by ehmry** [comment]
I have ``writeable`` policy support in ``ahci_drv`` and ``part_blk`` at 5da41f3e124d1bad45b1c816ab1c0214cb242c21. ``usb_drv`` was non-trivial and uses a static ``Block`` root component so I left it as-is.

**Comment by ehmry** [comment]
@chelmuth Rebased as read-only be default at bd45a761d8bf22a8da3c56f3a82b1c0e48f6e658.

**Comment by chelmuth** [comment]
Some libc_vfs run script produce errors like
```
[init -> ahci_drv] read-only session opened at device 0 for 'vfs -> 0'
```
It is also worthwhile to remove *libm* and *posix* from the libc_vfs tests as the test component does not depend on them, i.e., the libraries are not built and may not be available for integration.

**Comment by ehmry** [comment]
Rebased and fixed at d0b31187cc7cee42a4ddb8f2b04d284fcbc9654e

**Comment by chelmuth** [comment]
`make run/libc_vfs_fat KERNEL=...` still produces the error above, but is not in the autopilot list. Therefore, I merged the commit.
